src/utils/index.ts

```diff
@@ -17,4 +18,17 @@ const showToast = async (message: string) => {
       return toast.present();
 }
 
-export { showToast, hasError }
\ No newline at end of file
+const handleDateTimeInput = (dateTimeValue: any, inputType: string) => {
+  // TODO Handle it in a better way
+  // Remove timezone and then convert to timestamp
+  // Current date time picker picks browser timezone and there is no supprt to change it
+  let dateTime;
+  if (inputType === 'orderedBefore' || inputType === 'promisedBefore') {
```

Comment:
I think we should handle it differently, we should pass the input format and process date time accordingly

Reply:
Made the requested change